extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -614,6 +667,30 @@ export class SchemaCompareResult {
 		});
 	}
 
+	private createSourceAndTargetButtons(view: azdata.ModelView): void {
+		this.selectSourceButton = view.modelBuilder.button().withProperties({
+			label: '•••',
```

Comment:
did you take this from SSDT?

Reply:
no SSDT uses a dropdown, then opens a dialog. We decided to just have a button to open the dialog.